templates/gitserver/gitserver.StatefulSet.yaml

```diff
@@ -68,12 +64,20 @@ spec:
           name: config-file
         name: sg-config
       - name: repos
-        persistentVolumeClaim:
```

Comment:
Specifying `persistentVolumeClaim` here is now necessary, StatefulSet will refer to the claim below that has the same name as the volume (`repos`)